crates/store/re_data_source/src/data_source.rs

```diff
@@ -1,16 +1,15 @@
-use re_grpc_client::{ConnectionRegistryHandle, message_proxy};
+use re_grpc_client::ConnectionRegistryHandle;
 use re_log_types::{LogMsg, RecordingId};
 use re_smart_channel::{Receiver, SmartChannelSource, SmartMessageSource};
-use re_uri::RedapUri;
 
 use crate::FileContents;
 
 #[cfg(not(target_arch = "wasm32"))]
 use anyhow::Context as _;
 
-/// Somewhere we can get Rerun data from.
+/// Somewhere we can get Rerun logging data from.
 #[derive(Clone, Debug)]
-pub enum DataSource {
+pub enum LogDataSource {
```

Comment:
it's always been a log data source, it's just that the clarification until somewhat recently wasn't really necessary